OverlayPlugin.Core/MemoryProcessors/AtkStage/AtkStageMemory62.cs

```diff
@@ -12,8 +12,8 @@ interface IAtkStageMemory62 : IAtkStageMemory { }
 
     class AtkStageMemory62 : AtkStageMemory, IAtkStageMemory62
     {
-        // Offset to global for atkStage singleton instance. ghidra shows `14203c540`, base address of ffxiv_dx11.exe is `140000000`
-        private const int atkStageSingletonAddress = 0x203c540;
+        // Offset to global for atkStage singleton instance. ghidra shows `142044540`, base address of ffxiv_dx11.exe is `140000000`
+        private const int atkStageSingletonAddress = 0x2044540;
```

Comment:
Did you say this address was in FFXIVClientStructs? If so, could we fish it out of there?

Reply:
Yes, it's present, but it's in a fairly large yml file at `OverlayPlugin.Core/Thirdparty/FFXIVClientStructs/Global/ida/data.yml`, line 2278 in the current version:
```yml
# ...
classes:
# ...
  Component::GUI::AtkStage:
    instances:
      - ea: 0x142044540
```

I'm not sure what the best way would be to handle this. Package the yml in OP and extract at runtime? Bake it in at compile time? Pull the offset at compile time and put it in a json?

Reply:
Yeah, I'm not sure.  Lots of options.  Could even add "during the script conversion, copy the value into a cs file to use directly" as a possibility.  I'm not sure I have a strong feeling here, but it'd be nice to not have to update that manually too.